c2pa/claim-templates/claim_update.json

```diff
@@ -0,0 +1,33 @@
+{
+    "vendor": "starlinglab",
+    "recorder": "{{recorder}}",
+    "assertions": [
+        {
+            "label": "stds.schema-org.CreativeWork",
+            "data": {
+                "@context": "https://schema.org",
+                "@type": "CreativeWork",
+                "author": [
+                    {
+                        "@type": "{{author_type}}",
+                        "credential": [
+                        ],
+                        "identifier": "{{author_identifier}}",
+                        "name": "{{author_name}}"
+                    }
+                ]
+            }
+        },
+        {
+            "label": "c2pa.actions",
+            "data": {
+                "actions": [
+                    {
+                        "action": "c2pa.edited",
```

Comment:
This is a generic action saying we edited it and the changes aren't tracked in a C2PA-enabled tool.

Reply:
c2pa.edited should mean that the changes ARE tracked. There's c2pa.unknown for actions you can't identify.

Reply:
Got it. WIll change.